crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -22,30 +22,20 @@
 //! ## Configure the network.
 //! network:
 //!
-//!   ## Use the Holo-provided default production bootstrap server.
-//!   bootstrap_service: https://bootstrap.holo.host
+//!   ## Use the Holochain-provided dev-test bootstrap server.
+//!   bootstrap_url: https://devtest-bootstrap-1.holochain.org
 //!
-//!   ## This currently has no effect on functionality but is required. Please just include as-is for now.
-//!   network_type: quic_bootstrap
+//!   ## Use the Holochain-provided dev-test sbd/signalling server.
+//!   signal_url: wss://devtest-sbd-1.holochain.org
```

Comment:
Note, this is moving us in the direction of not hosting production services, but iirc we did not complete that discussion. @zippy @ThetaSinner